### DISCUSSION TITLE
Durchlaufende Posten

### DISCUSSION THREAD

**Discussion by StephanKuehn** [stephanstapel/ZUGFeRD-csharp#463] [Q&A] [answered]
Ich muss durchlaufende Posten (hauptsächlich amtliche Gebühren, die im Namen und Auftrag eines Kunden entrichtet worden sind)
abbilden und tue mich etwas schwer damit, die passende Kodierung für die USt zu finden.

```
desc.AddTradeLineItem(
        lineID: "02", 
        name: "Gebühren",
        description: "Amtliche  Gebühren", 
        unitCode: QuantityCodes.C62, 
        => taxType: TaxTypes.VAT,
        => taxPercent: 0, 
        netUnitPrice: 98.62m,
        unitQuantity: 1,
        grossUnitPrice: 98.62m,
        billedQuantity: 1,
        lineTotalAmount: 98.62m,
        => categoryCode: TaxCategoryCodes.E);
```
Wäre das so korrekt?

Wo finde ich generell die Erläuterung zu den Feldern in ZUGFeRD?

Viele Grüße
Stephan

**Comment by stephanstapel** [comment]
Hallo @StephanKuehn,
die Beschreibung für die Felder findest Du teilweise in der XML-Dokumentation bzw. dem Tooltip beim Programmieren in Visual Studio.
Die Werte sind im Quellcode meistens so gut beschrieben wie sie in der offizielenn Doku beschrieben sind:

https://github.com/stephanstapel/ZUGFeRD-csharp/blob/master/ZUGFeRD/TaxCategoryCodes.cs

Deine Parameter finde ich so plausibel. Sofern die Validierer nichts monieren, ist das bestimmt ok.
Leider sind die definierten  Regeln sehr strikt, meines Erachtens oft zu strikt.

Hattest Du den Code gesehen, der wäre ggf. auch passend:

https://github.com/stephanstapel/ZUGFeRD-csharp/blob/ad1b165ed79b5e61591d81934587575841b3f757/ZUGFeRD/TaxCategoryCodes.cs#L179

den finanztechnischen Hintergrund dieses Codes verstehe ich nicht, könnte aber von der Beschreibung her auch passen:

https://github.com/stephanstapel/ZUGFeRD-csharp/blob/ad1b165ed79b5e61591d81934587575841b3f757/ZUGFeRD/TaxCategoryCodes.cs#L193

**Comment by StephanKuehn** [comment]
Hallo @stephanstapel,

vielen Dank für die schnelle Antwort.

Ich denke, das ist letztendlich eine Frage für den Steuerberater. 

_Ich würde momentan zu TaxCategoryCodes.O tendieren:
Durchlaufende Posten sind Ausgaben, die ein:e Unternehmer:in im Auftrag eines oder einer anderen Unternehmer:in tätigt. Es handelt sich um Kosten, die der oder die Auftraggeber:in selbst trägt und die der oder die Auftragnehmer:in lediglich weiterleitet. ... Voraussetzung für das Vorliegen eines umsatzsteuerlichen durchlaufenden Postens ist, dass der oder die Unternehmer:in an dem zivilrechtlichen Vertrag nicht beteiligt ist, also weder Gläubiger:in noch Schuldner:in der Leistung ist. (https://ecovis-kso.com/blog/umsatzsteuer-durchlaufende-posten-was-ist-das-eigentlich/)_

TaxCategoryCodes.E bezieht sich in Deutschland wohl auf _§4 UStG Steuerbefreiungen bei Lieferungen und sonstigen Leistungen_,
TaxCategoryCodes.Z bezieht sich meiner Ansicht nach auf Deutschland, in anderen Ländern scheint es einen USt-Satz von 0% bspw. auf bestimmte Lebensmittel oder Sanitärprodukte zu geben.

Viele Grüße
Stephan

**Comment by stephanstapel** [comment]
Poste doch dann seine Antwort hier bitte, dann haben andere bestimmt auch noch was davon!

**Comment by lagset** [comment, marked as answer]
Hatte das gleiche Problem und entsprechend recherchiert. Habe hier noch keine "offizielle" Antwort, jedoch sollte auf Basis meiner Recherche für Tax Category Code "E" in Kombination mit dem Exemption Reason Code (BT-121) "VATEX-EU-79-C" verwendet werden.

Der Vatex Code kommt aus folgender Liste des CEF. die für BT-121 verpflichtend ist: https://ec.europa.eu/digital-building-blocks/sites/display/DIGITAL/Registry+of+supporting+artefacts+to+implement+EN16931?preview=/467108974/467108990/VAT

Der Code bezieht sich laut [Peppol BIS 3.0](https://docs.peppol.eu/poacc/billing/3.0/codelist/vatex/) auf "article 79, point c of Council Directive 2006/112/EC".

Liest man in der Richtline der EU (https://eur-lex.europa.eu/legal-content/DE/TXT/?uri=CELEX:32006L0112) in der deutschen Fassung nach, wird hier unter Artikel 79 c) direkt auf "durchlaufende Posten" Bezug genommen:

![image](https://github.com/user-attachments/assets/8300eb6f-4d65-4e9d-a5c2-c54f0b1416b6)

**Wieso nicht Code "O"?**
Da die Business Rule BR-O-11 (und folgende) vorschreibt, dass es im Falle von "O" nur einen einzigen Breakdown auf der Rechnung geben darf, wäre eine Abbildung eines durchlaufenden Posten nicht sinnvoll möglich, da dieser auch innerhalb einer Rechnung in Kombination mit Positionen vorkommen kann und darf, die USt. enthalten.
